or-grel/src/test/java/com/google/refine/expr/functions/strings/SmartSplitTests.java

```diff
@@ -96,26 +79,8 @@ public void testSmartSplitCharSepSpace() {
     @Test
     public void testSmartSplitStringSepGiven() {
         String testString = "teststring1#@$teststring2#@$teststring3#@$teststring4";
-        String[] expected = { "teststring1", "teststring2", "teststring3", "teststring4" };
-        String[] actual = (String[]) invoke("smartSplit", testString, "#@$");
-        assertEquals(expected, actual);
-    }
-
-    /**
-     * Lookup a control function by name and invoke it with a variable number of
-     * args
-     */
-    private static Object invoke(String name, Object... args) {
-        // registry uses static initializer, so no need to set it up
-        Function function = ControlFunctionRegistry.getFunction(name);
-        if (function == null) {
-            throw new IllegalArgumentException("Unknown function " + name);
-        }
-        if (args == null) {
-            return function.call(bindings, new Object[0]);
-        } else {
-            return function.call(bindings, args);
-        }
+        Object actual = invoke("smartSplit", testString, "#@$");
+        assertTrue(actual instanceof EvalError);
```

Comment:
For me that was a conscious and intended change: from my perspective multi-char support for the smartSplit function is mostly a side-effect of us wanting to support multi-char CSV support at the import stage, but I don't think it is critical in this GREL context. It's fair to disagree, and I'd be happy to restore this behaviour via commons-csv for instance. For sure, I should have done a better job at announcing this change, but honestly I think in the context of such large-scale architecture changes we need to have a proportionate approach and accept small differences in functionality - otherwise we are just making it impossible for ourselves to do significant renovation work on the code base.

Reply:
I have rebased my changes on top of the master branch, after your univocity migration, so that's resolved.